discovery/README.md

```diff
@@ -19,6 +15,10 @@ use across multiple organizations. It should allow discovering of machines
 and/or services running somewhere. When exactly an SD is popular enough to
 justify being added to Prometheus natively is an open question.
 
+Note: As part of lifting the past moratorium on new SD implementations it was
```

Comment:
i.e.

Do we need to even mention the moratorium here? It's historical now.

Reply:
I think there's value in keeping mention of it for some period of time. Even if we had an announcement on the mailing list not everyone's going to read it, explicitly saying the moratorium is over and there's a new requirement for new SD implementations is clearer.

Reply:
You still need to fix i.e.